pandas/core/groupby.py

```diff
@@ -3468,6 +3459,24 @@ def _apply_to_column_groupbys(self, func):
              in self._iterate_column_groupbys()),
             keys=self._selected_obj.columns, axis=1)
 
+    def count(self):
```

Comment:
this needs to take the ``axis`` kw to be back-compat. (I see that you changed a test to fix this, pls change back).